### DISCUSSION TITLE
Raspberry pi 3 Model B V1.2

### DISCUSSION THREAD

**Discussion by Exo84** [camueller/SmartApplianceEnabler#370] [Installation und Betrieb]
Hallo,

ich habe eine ältere Respberry "rumliegen" und würde diese gerne für den SAE verwenden.
Sollte es mit o.g. Model funktionieren?

Danke
Gruß

**Comment by camueller** [comment]
Ja, alles ab Pi 2 ist sinnvoll nutzbar.

**Reply by Exo84** [reply]
Danke!
